osu.Game/Localisation/EditorDialogsStrings.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class EditorDialogsStrings
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.EditorDialogs";
+
+        /// <summary>
+        /// "Would you like to create a blank difficulty?"
+        /// </summary>
+        public static LocalisableString NewDifficultyDialogHeader => new TranslatableString(getKey(@"new_difficulty_dialog_header"), @"Would you like to create a blank difficulty?");
+
+        /// <summary>
+        /// "Yeah, let&#39;s start from scratch!"
+        /// </summary>
+        public static LocalisableString CreateNew => new TranslatableString(getKey(@"create_new"), @"Yeah, let's start from scratch!");
+
+        /// <summary>
+        /// "No, create an exact copy of this difficulty"
+        /// </summary>
+        public static LocalisableString CreateCopy => new TranslatableString(getKey(@"create_copy"), @"No, create an exact copy of this difficulty");
+
+        /// <summary>
+        /// "I changed my mind, I want to keep editing this difficulty"
+        /// </summary>
+        public static LocalisableString KeepEditing => new TranslatableString(getKey(@"keep_editing"), @"I changed my mind, I want to keep editing this difficulty");
```

Comment:
Before making these localisable, I checked this dialog UX, and not sure if it's me, but it's overly complex / confusing:
- you have to press the current ruleset to copy and pressing other rulesets makes a new beatmap and is just inconsistent (i.e. the copying function is hidden)
- there are two "no" answers to this dialog and if people were skimming, they will just read the "yes" and "no" and not the rest
- you are greeted with another dialog if you have pending changes to the current beatmap

We can get rid of this dialog completely and the issues above if it copying was just split to another menu item like google docs:
![image](https://user-images.githubusercontent.com/35318437/212504414-cf73803a-5c04-46f3-88c0-2eb82177cf17.png)

Reply:
> you have to press the current ruleset to copy and pressing other rulesets makes a new beatmap and is just inconsistent (i.e. the copying function is hidden)

This was done this way because stable did it this way. Not necessarily opposed to changing it, just something to keep in mind.

> you are greeted with another dialog if you have pending changes to the current beatmap (negating what is selected in the previous dialog if "oops, continue editing" was selected)

Don't you mean the other way around? I believe the prompt to save the beatmaps comes second. And this is an issue regardless of whether copy is a separate button or not, no?

Reply:
> Don't you mean the other way around? I believe the prompt to save the beatmaps comes second.

I may have worded it badly, but the save beatmap dialog comes second, yes.

> And this is an issue regardless of whether copy is a separate button or not, no?

No, my issue was more that you have to interact with multiple dialogs in that case instead of usually one.

No harm in leaving this, just as a consideration for the future.